lib/decompress/zstd_decompress.c

```diff
@@ -80,20 +80,33 @@ typedef enum { ZSTDds_getFrameHeaderSize, ZSTDds_decodeFrameHeader,
 typedef enum { zdss_init=0, zdss_loadHeader,
                zdss_read, zdss_load, zdss_flush } ZSTD_dStreamStage;
 
+
+typedef struct {
+    U32 fastMode;
+    U32 tableLog;
+} ZSTD_seqSymbol_header;
+
+typedef struct {
+    U16  nextState;
+    BYTE nbAdditionalBits;
+    BYTE nbBits;
+    U32  baseValue;
+} ZSTD_seqSymbol;
+
 typedef struct {
-    FSE_DTable LLTable[FSE_DTABLE_SIZE_U32(LLFSELog)];
-    FSE_DTable OFTable[FSE_DTABLE_SIZE_U32(OffFSELog)];
-    FSE_DTable MLTable[FSE_DTABLE_SIZE_U32(MLFSELog)];
+    ZSTD_seqSymbol LLTable[FSE_DTABLE_SIZE_U32(LLFSELog)];
```

Comment:
Is `FSE_DTABLE_SIZE_U32(LLFSELog)` still the correct size? We'll probably want a new macro

```c
#define ZSTD_SEQTABLE_SIZE(FSELog) FSE_DTABLE_SIZE_U32((FSELog))
```

Reply:
Yes, it's the correct size.
Yes, adding a new macro is a nonetheless cleaner way to do this.